### DISCUSSION TITLE
We use custom remote schema loader and sometimes call to custom loader is not happening

### DISCUSSION THREAD

**Discussion by thulaban1990** [networknt/json-schema-validator#1135] [Q&A]
Hi,

We use custom schema loader and sometimes call to custom loader is not happening based on logs in the custom schema loader.
is this because schemas gets cached?

how to verify that this is because of cache if that was the root cause?
How to disable cache or how to control the cache? as if change schema details from remote, we expect updated schema to be retrieved. So, based on cache eviction time or some parameter could help.

We are fine to disable cache as well. I see a flag called enableUriSchemaCache however I dont see any documentation on how to use it and its default behavior.

Could someone post how to debug this issue and how to fix it?

Thank you,
Siva

**Comment by thulaban1990** [comment]
Hi, can anyone provide how to fix this?

**Comment by thulaban1990** [comment]
@stevehu could you provide any help?

**Comment by stevehu** [comment]
I am not able to understand your issue. Could you please create a test case to replicate the issue? Without knowing your schema and the JSON object, there is no way for us to figure out the issue you are facing.
